pumpkin-core/src/text/color.rs

```diff
@@ -59,7 +95,8 @@ impl Color {
                 NamedColor::Yellow => text.bright_yellow(),
                 NamedColor::White => text.white(),
             },
-            Color::Rgb(_) => todo!(),
```

Comment:
I will not agree to making the colors not work in the terminal, This would also then just panic